app/controllers/conference_registrations_controller.rb

```diff
@@ -1,10 +1,11 @@
 class ConferenceRegistrationsController < ApplicationController
   before_filter :authenticate_user!, except: [:new, :create]
   load_resource :conference, find_by: :short_title
-  authorize_resource :conference_registrations, class: Registration
+  authorize_resource :conference_registrations, class: Registration, except: [:new, :create]
   before_action :set_registration, only: [:edit, :update, :destroy, :show]
 
   def new
+    authorize! :new, Registration.new(conference_id: @conference.id)
```

Comment:
At the end of the action we define @registration, perhaps we should simply authorize that?